swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    """Child class that inherits attributes and methods from Item class as parent class:"""
+
+
+    def __init__(self, condition = 0, age = 0):
+        """Setting category attribute as default with "Clothing" and inherits attributes from Item class."""
+        super().__init__(category= "Clothing", condition = condition, age = age)
```

Comment:
Great use of `super`! And very clever to see we don't need `category` as an argument for the subclass constructor.